usaspending_api/search/v2/views/enums.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class SpendingLevel(Enum):
```

Comment:
Really like the use of the Enum in a central location for the `search` app, we should use them much more across the codebase.